[2024-08-25T08:42:18.796+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-25T08:42:18.829+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_catchup_and_backfill_v2.task1 backfill__2024-08-20T00:00:00+00:00 [queued]>
[2024-08-25T08:42:18.848+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_catchup_and_backfill_v2.task1 backfill__2024-08-20T00:00:00+00:00 [queued]>
[2024-08-25T08:42:18.852+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 6
[2024-08-25T08:42:18.887+0000] {taskinstance.py:2879} INFO - Executing <Task(BashOperator): task1> on 2024-08-20 00:00:00+00:00
[2024-08-25T08:42:18.895+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1806) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-25T08:42:18.899+0000] {standard_task_runner.py:72} INFO - Started process 1828 to run task
[2024-08-25T08:42:18.911+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_catchup_and_backfill_v2', 'task1', 'backfill__2024-08-20T00:00:00+00:00', '--job-id', '81', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_catchup_and_backfill.py', '--cfg-path', '/tmp/tmpam8j9s6n']
[2024-08-25T08:42:18.918+0000] {standard_task_runner.py:105} INFO - Job 81: Subtask task1
[2024-08-25T08:42:19.002+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_catchup_and_backfill_v2.task1 backfill__2024-08-20T00:00:00+00:00 [running]> on host ae952724060e
[2024-08-25T08:42:19.120+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='shreyas' AIRFLOW_CTX_DAG_ID='dag_with_catchup_and_backfill_v2' AIRFLOW_CTX_TASK_ID='task1' AIRFLOW_CTX_EXECUTION_DATE='2024-08-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='backfill__2024-08-20T00:00:00+00:00'
[2024-08-25T08:42:19.122+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-25T08:42:19.149+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-08-25T08:42:19.152+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo This is a simple bash command!']
[2024-08-25T08:42:19.167+0000] {subprocess.py:86} INFO - Output:
[2024-08-25T08:42:19.169+0000] {subprocess.py:93} INFO - This is a simple bash command!
[2024-08-25T08:42:19.170+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-08-25T08:42:19.212+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-25T08:42:19.213+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_catchup_and_backfill_v2, task_id=task1, run_id=backfill__2024-08-20T00:00:00+00:00, execution_date=20240820T000000, start_date=20240825T084218, end_date=20240825T084219
[2024-08-25T08:42:19.253+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-25T08:42:19.270+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-25T08:42:19.272+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
